### PR TITLE
Kemalyst crystal pg full

### PR DESCRIPTION
**Pull request by drujensen**

### CODE REVIEW

**Comment by nbrady-techempower** [comment]
@drujensen Thanks for the pull request!

Would you mind removing some of the files that aren't necessary for running the tests like the Docker stuff and css files.

Also, we can't have the 5mb kemalyst binary here. That should be pulled in at test time and added to the .gitignore. 

Let me know if there's anything else I can help with!

**Comment by drujensen** [comment]
@nbrady-techempower done.  thanks!

File: frameworks/Crystal/kemalyst/config/database.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+pg:
+  database: postgres://benchmarkdbuser:benchmarkdbpass@127.0.0.1/hello_world
```
Comment (nbrady-techempower) [inline frameworks/Crystal/kemalyst/config/database.yml]: you can replace `127.0.0.1` with `TFB-database` which allows you to remove the `sed` statement in the `setup.sh`

**Comment by drujensen** [comment]
@nbrady-techempower done

**Comment by nbrady-techempower** [comment]
@drujensen Thanks! Looks good!

**Comment by drujensen** [comment]
Nate, is there a way to find out how kemalyst is ranked before the next official round is released?  I am wondering if the reuse_port is going to fix the parallism issue we encountered in Kemal.

**Comment by nbrady-techempower** [comment]
@drujensen There will be several preview rounds that will have logs and data. Usually it's announced on the Google Group. Hoping to get one out this week.

**Comment by nbrady-techempower** [comment]
@drujensen 

Kemalyst is currently breaking. Looks like it has to do with this repo: https://github.com/kemalyst/kemalyst-model.git 0.6.0 not being available.

**Pull request merged**